### DISCUSSION TITLE
Personal Xiao seeed studio with GC9A01

### DISCUSSION THREAD

**Discussion by EmrysCES** [Bodmer/TFT_eSPI#3166] [General]
Dear Bodmer,

Thanks for the previous help! I recently watched this video:
https://www.youtube.com/watch?v=mqSe_uMpxIs&pp=ygUVdGhlbGFzdG91dHBvc3R3b2tzaG9w

The screen worked perfectly but I am now looking for a smaller chip. I tought the Xiao Seed studio ESP32_C3 chip would work well, but I am new to all of this. The things i want are pretty difficult and i think it would take a long time for you to figure out, so please think twice before helping me beacause they may be impossible.

This is what i want:

The complete wireing from the Aliexpress GC9A01 screen with the Xiao Seed studio ESP32_C3 chip.
The code in Arduino IDE for loading GIF files in the form of C header files onto the chip, similar to the video
Again, i know this is a lot of work, so feel free to just ignore my request if you feel like, but please tell me if you decide to not take on the task. It is pretty difficult and if you dont know how, would you please help me finding someone who has the skills neccesary?

Thank you for considering to help me and I would be very happy If you decide to help me.
Greetings,

EmrysCES

**Comment by Bodmer** [comment]
Try the user setup file 70c and adapt for your display:
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup70c_ESP32_C3_ILI9341.h
